src/pep8ext_naming.py

```diff
@@ -6,6 +6,7 @@
 from fnmatch import fnmatchcase
 from functools import partial
 from itertools import chain
+from typing import List, Tuple
```

Comment:
I think we can also drop Python 3.8 support, which has reached the end of his support life. I'll do that in a separate pull request.

Reply:
#238